packages/dx-react-chart/docs/guides/zoom-and-pan.md

```diff
@@ -0,0 +1,31 @@
+# React Chart - Zoom and Pan
+
+Users can zoom and pan (scroll) the React Chart. These actions change the chart's viewport. Mouse and touch gestures are supported.
+
+## Basic Setup
+
+Import the [ZoomAndPan](../reference/zoom-and-pan.md) plugin and add it to the `Chart` component to enable zooming and panning.
+
+Note that the [plugin order](./plugin-overview.md#plugin-order) is important.
+
+### Uncontrolled Mode
+
+In uncontrolled mode, add the `ZoomAndPan` plugin to the `Chart` component.
+
+.embedded-demo({ "path": "zoom-pan/uncontrolled", "showThemeSelector": true })
+
+### Controlled Mode
+
+In controlled mode, pass the viewport's boundaries to the `ZoomAndPan` plugin's `viewport` property and handle `onViewportChange` event to control zoom and pan externally.
```

Comment:
"to control zoom and pan externally" What do you mean by 'externally'?

Reply:
This is a React thing. Components support uncontrolled and controlled modes. In uncontrolled mode, the component's state is manged by the component itself. In controlled mode, the component's state is controlled from outside, hence "controlled" mode.

Reply:
```suggestion
In controlled mode, pass the viewport's boundaries to the `ZoomAndPan` plugin's `viewport` property and handle `onViewportChange` event to control zoom and pan.
```